frontend/www/js/omegaup/components/user/BasicInformationEdit.vue

```diff
@@ -56,6 +56,7 @@
       <omegaup-datepicker
         v-model="birthDate"
         :required="false"
+        :max="new Date().toISOString().split('T')[0]"
```

Comment:
```suggestion
        :max="new Date()"
```

the parameter must be a `Date` object according to the declaration in `DatePicker.vue` L32. the provided expression is not one, so the only possible thing that is a `Date` object is just `new Date()`.

Reply:
In fact, I used new Date() at first, but it doesn't work. It turns out that the element only accepts dates in ISO format. That's why I intentionally convert it to ISOString.
A blog on [stackoverflow.com](https://stackoverflow.com/questions/38638424/html5-input-type-date-disable-dates-before-today) also explains this.

Reply:
this still needs to happen.